3rdparty/openvx/include/ivx.hpp

```diff
@@ -1376,6 +1389,25 @@ class Image : public RefWrapper<vx_image>
     static Image createUniform(vx_context context, vx_uint32 width, vx_uint32 height, vx_df_image format, const void* value)
     { return Image(vxCreateUniformImage(context, width, height, format, value)); }
 #endif
+    template <typename T>
+    static Image createUniform(vx_context context, vx_uint32 width, vx_uint32 height, const T value)
```

Comment:
IMHO, guessing image format from the value type is not convenient and fragile idea!
I suggest adding `vx_df_image format` argument.
BTW, this method is implemented by me as well in the PR I haven't yet submitted :)